### ISSUE TITLE
[Supporting the platform – Make Code – Import] Focus indicator is not visible on multiple controls after applying high contrast (black/white) theme.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3565]
**User Experience:**
If the desired component, on which keyboard operations will interact at any point in time is not visually determined, people with attention limitations, short term memory limitations, or limitations in executive processes will find it difficult to discover where the focus is located in high contrast theme.

**Note**: User credentials should NOT be included in the bug.

**Pre-requisites:**
Enable high contrast mode. Settings->Ease of Access->Color & High contrast

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Verify the focus visibility on controls like(Close, Import File, Import URL, Your GitHub repo) after applying high contrast black/white theme.

**Actual Result:**
Keyboard focus indicator is not visible on controls like(Close, Import File, Import URL, Your GitHub repo) under 'Import' pop-up and due to this, visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus indicator should be visible on controls like(Close, Import File, Import URL, Your GitHub repo) under 'Import' pop-up, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Note:**
1. Same issue exists under 'Import File' pop-up window. Path: "Import" button and select it->"Import" dialog->Select Import file option->Import file pop-up.
2. Same issue exists under 'Import URL' pop-up window. Path: "Import" button and select it->"Import" dialog->Select Import URL option->'Open project URL' pop-up.
3. Same issue exists under 'Sign in with GitHub' pop-up window. Path: "Import" button and select it->"Import" dialog->Select Your GitHub Repo option->'Sign in with GitHub' pop-up.
4. Same issue exists throughout the application.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev1, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.3.1, #Win10-Edge(Chromium)

[WCAG4.3.1_Import_HC.zip](https://github.com/microsoft/pxt-microbit/files/5480013/WCAG4.3.1_Import_HC.zip)
